voltixApp/VoltixApp/View Models/SendCryptoViewModel.swift

```diff
@@ -69,7 +69,8 @@ class SendCryptoViewModel: ObservableObject, TransferViewModel {
             chainSpecific: BlockChainSpecific.UTXO(byteFee: BigInt(tx.feeInSats)),
             utxos: utxoInfo,
             memo: tx.memo,
-            swapPayload: nil
+            swapPayload: nil, 
+            vaultPubKeyECDSA: vault.pubKeyECDSA
         )
         
         if let vault = ApplicationState.shared.currentVault {
```

Comment:
since we pass `vault` into this method now , so we don't need to take vault from ApplicationState.shared.currentVault anymore , could you please update this as well?